symplectic_manifolds.tex

```diff
@@ -70,24 +103,52 @@ \section{Symplectic vector spaces}
 Even if, as noted before, the choice of such a basis is far from unique, we'll refer to it as a \emph{canonical} basis.
 
 \begin{definition}
-	A subspace $W$ of a symplectic vector space $(V, \Omega)$ is said to be \textbf{symplectic} as well if $(W, \Omega\vert_W)$ is a symplectic vector space. If instead $\Omega\vert_W = 0$, the subspace is called \textbf{isotropic} and its $\Omega$-orthogonal is then called \textbf{coisotropic}. When a subspace is both, it's named \textbf{Lagrangian}.
+	A subspace $W\subset V$ of a symplectic vector space $(V, \Omega)$ is said to be \textbf{symplectic} as well if $(W, \Omega\vert_W)$ is a symplectic vector space. If instead $\Omega\vert_W(u,v) = 0$ $\forall u,v\in W$, the subspace is called \textbf{isotropic}.
+	% and its $\Omega$-orthogonal is then called \textbf{coisotropic}. When a subspace is both, it's named \textbf{Lagrangian}.
 \end{definition}
 
+\begin{example}
+	Let $V=\langle e_1,\dots,e_n,f_1,\dots,f_n\rangle$ where $e_1,\dots,e_n,f_1,\dots,f_n$ is the canonical basis. Then
+	\begin{enumerate}
+		\item	$W=\langle e_{i_1},\dots,e_{i_k},f_{i_1},\dots,f_{i_k}\rangle$ where $k\leq n$, $1\leq i_s\leq n$ and $i_s\neq i_{s'}$ if $s\neq s'$, is a symplectic subspace of V.
+		\item	$W=\langle e_{i_1},\dots,e_{i_k}\rangle$ where $k\leq n$, is a isotropic subspace of V
+	\end{enumerate}
+\end{example}
+
 Thus the decomposition~\eqref{eq:symp_decomp} (when $U = 0$, so $\Omega$ is non-degenerate) is a decomposition of $V$ in minimal symplectic subspaces.
 
 \begin{definition}
-	A \textbf{symplectomorphism} is a linear map $\phi : (V, \Omega) \to (V', \Omega')$ between symplectic vector spaces $(V, \Omega)$ and $(V', \Omega')$ such that
+	A \textbf{symplectomorphism} of symplectic vector spaces $(V, \Omega)$ and $(V', \Omega')$ is a linear isomorphism $\phi : V\overset\sim\longmapsto V' $  such that
 	\begin{eqalign}
-		\Omega'(\phi(u), \phi(v)) = \Omega(u,v), \quad \forall u,v \in V,
+		\phi^* \Omega' = \Omega
 	\end{eqalign}
-	or, alternatively, if $\phi^* \Omega' = \Omega$.
+	where $\phi*$ denotes the pull-back (recall $(\phi^* \Omega')(u,v):=\Omega'(\phi(u), \phi(v))$).
+	If such a $\phi$ exists, $(V, \Omega)$ and $(V', \Omega')$ are called \textbf{symplectomorphic}. 
 \end{definition}
 
-Hence, in the case of symplectic vector spaces, Theorem~\ref{th:decomp_th} says that
+This is equivalent of requiring that following diagram commutes:
+\begin{figure}[H]
+\centering
+	\include{figures/diag-symplectomorphic}
+\end{figure}
+\vspace{-1cm}
+
+\begin{remark}
+The requirement that $\phi$ is an isomorphism is not restrictive, since this is a simple consequence of the trivial kernel of $\Omega$.
```

Comment:
Indent!

Reply:
Also this is not really true, you should say that the \phi is forced to be an isomorphism by the compatibility condition with the symplectic forms.